lms/envs/common.py

```diff
@@ -130,13 +130,11 @@
     # Enables ability to restrict enrollment in specific courses by the user account login method
     'RESTRICT_ENROLL_BY_REG_METHOD': False,
 
-    # Enables the LMS bulk email feature for course staff
-    'ENABLE_INSTRUCTOR_EMAIL': True,
-    # If True and ENABLE_INSTRUCTOR_EMAIL: Forces email to be explicitly turned on
+    # If True and BulkEmailFlag.is_enabled(): Forces email to be explicitly turned on
     #   for each course via django-admin interface.
-    # If False and ENABLE_INSTRUCTOR_EMAIL: Email will be turned on by default
+    # If False and BulkEmailFlag.is_enabled(): Email will be turned on by default
     #   for all Mongo-backed courses.
-    'REQUIRE_COURSE_EMAIL_AUTH': True,
+    'REQUIRE_COURSE_EMAIL_AUTH': False,
```

Comment:
For some reason, this value is affecting local bok choy tests despite `REQUIRE_COURSE_EMAIL_AUTH` being `False` in `acceptance.py`